### ISSUE TITLE
Error inverse_normalization_matrix = torch.inverse(normalization_matrix)

### ISSUE THREAD

**Issue by Yaltay** [albumentations-team/autoalbument#54]
Hello, I received this error and I can't fix it. I wonder what the source of this error is. My search.yml file is like this.

_version: 2

task: semantic_segmentation

policy_model:
  task_factor: 0.1
  gp_factor: 10
  temperature: 0.05
  
  num_sub_policies: 15  # 25 → 15 (binary segmentation için)
  
  num_chunks: 4
  
  operation_count: 3  # 4 → 3 (daha conservative)

semantic_segmentation_model:
  _target_: autoalbument.faster_autoaugment.models.SemanticSegmentationModel
  
  num_classes: 2  # 19 → 2 (background + tumor)
  
  architecture: Unet  
  encoder_architecture: resnet34  # resnet50 → resnet34 (daha hızlı)
  
  pretrained: true

data:
  dataset:
    _target_: dataset.BoneTumorSearchDataset
    images_dir: path/path/path
    masks_dir:  path/path/path

  input_dtype: float32  # uint8 → float32 (normalized data için)

  preprocessing:
    - Resize:
        height: 512
        width: 512
        interpolation: 3 
  
  normalization:
    mean: [0.485, 0.456, 0.406]  # Veya [0.5, 0.5, 0.5]
    std: [0.229, 0.224, 0.225]   # Veya [0.25, 0.25, 0.25]

  dataloader:
    _target_: torch.utils.data.DataLoader
    batch_size: 16  
    shuffle: true
    num_workers: 0  
    pin_memory: true
    drop_last: true

optim:
  main:
    _target_: torch.optim.Adam
    lr: 1e-3  
    betas: [0.9, 0.999]  

  policy:
    _target_: torch.optim.Adam
    lr: 1e-3
    betas: [0.9, 0.999]  

seed: 42

hydra:
  run:
    dir: ${config_dir:}/bone_tumor_outputs/${now:%Y-%m-%d}/${now:%H-%M-%S}
   

trainer:

  max_epochs: 30  # 50 → 30 (kemik tümörü için yeterli)
  
  benchmark: true
  
  
  gpus: 1  # GPU varsa 1, yoksa 0
  
  
  
  default_root_dir: checkpoints/
  

  

  check_val_every_n_epoch: 1
